modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/HaskellHttpClientCodegen.java

```diff
@@ -681,6 +685,9 @@ private void processReturnType(CodegenOperation op) {
                 op.vendorExtensions.put(X_HAS_UNKNOWN_RETURN, true);
             } else {
                 returnType = "NoContent";
+                if(!op.vendorExtensions.containsKey(X_INLINE_ACCEPT)) {
```

Comment:
Minor suggestion: adding a space after `if`

Reply:
thanks, will use auto code-formatter to fix